### PR TITLE
feat: refact project using rollup for npm module using

### PR DESCRIPTION
**Pull request by yugasun**
Now this project can not be used for npm module, so I refactor this project using rollup, and bundle to [Commonjs](https://en.wikipedia.org/wiki/CommonJS) and ES module standard.

After merging this pr, you can `npm run build` and `npm publish`, then you can publish this module to [npmjs.com](https://www.npmjs.com/)

### CODE REVIEW

File: test/test-payment.js [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
+const {
+    Account,
+    Blockchain,
+    constants,
+    Convert,
+} = require('../dist/index');
 
 /*======= Change the below before run ==========*/
-const SEED = "<please input your seed phrase>";
+const SEED = bip39.generateMnemonic();
```
Comment (SheldonYS) [inline test/test-payment.js]: If we random generate mnemonic in every test, the balance would always be 0 and fail to do the payment.

**Comment by SheldonYS** [comment]
`src/utils/axlsign.js` can be deleted. We changed to use `package.json` to download this lib. (see https://github.com/virtualeconomy/js-v-min-sdk/commit/b1a03801c5baa18f50255e08221d1bd77861dcd1 )

File: test/test-payment.js (hunk 1 above)
Comment (yugasun) [reply test/test-payment.js]: Maybe you can add unit test using [Jest](https://jestjs.io/) for this project.

**Comment by yugasun** [comment]
> `src/utils/axlsign.js` can be deleted. We changed to use `package.json` to download this lib. (see [b1a0380](https://github.com/virtualeconomy/js-v-min-sdk/commit/b1a03801c5baa18f50255e08221d1bd77861dcd1) )

Because we can not find github project for `axlsign`(although the source is [curve25519-js](https://github.com/wavesplatform/curve25519-js), but they did not publish), so I preferred to keep  it in project.

File: test/test-payment.js (hunk 1 above)
Comment (SheldonYS) [reply test/test-payment.js]: Thanks for your suggestion. Full function version `js-v-sdk` is using mocha to do the unit test. This repo will also be changed when we have time to maintain.
Screenshot 2019-07-26 at 11 35 27 AM

Comment (yugasun) [reply test/test-payment.js]: That's great~

**Comment by SheldonYS** [comment]
> > `src/utils/axlsign.js` can be deleted. We changed to use `package.json` to download this lib. (see [b1a0380](https://github.com/virtualeconomy/js-v-min-sdk/commit/b1a03801c5baa18f50255e08221d1bd77861dcd1) )
> 
> Because we can not find github project for `axlsign`(although the source is [curve25519-js](https://github.com/wavesplatform/curve25519-js), but they did not publish), so I preferred to keep it in project.

OK. Keep the file makes sense.

File: src/utils/transaction.js [hunk 2]
```diff
@@ -57,84 +64,152 @@ function getFields(type) {
...
+    let byteArr = Convert.stringToByteArray(description);
+
+    let typeArr = new Array(1);
+    typeArr[0] = SHORTTEXT_TYPE;
+
+    let length = byteArr.length;
+    let lengthArr = Convert.shortToByteArray(length);
+
+    return typeArr.concat(lengthArr.concat(byteArr));
+}
```
Comment (SheldonYS) [inline src/utils/transaction.js]: The method `transferContract`, `transferAmount`, `transferAccount`, `transferShortTxt` are not used in `js-v-min-sdk` (they are smart contract feature, which used in another repo named `js-v-sdk`). We can delete them here.

File: src/utils/converters.js [hunk 3]
```diff
@@ -1,283 +1,327 @@
...
- * holder information and the developer policies on copyright and licensing.  *
- *                                                                            *
- * Unless otherwise agreed in a custom licensing agreement, no part of the    *
- * Nxt software, including this file, may be copied, modified, propagated,    *
- * or distributed except according to the terms contained in the LICENSE.txt  *
- * file.                                                                      *
- *                                                                            *
- * Removal or modification of this copyright notice is prohibited.            *
- *                                                                            *
- ******************************************************************************/
```
Comment (zl730) [inline src/utils/converters.js]: the copyright should be kept

Comment (yugasun) [reply src/utils/converters.js]: No, must add the copyright.

**Pull request merged**